### PR TITLE
Updated compilation / test bash scripts

### PR DESCRIPTION
**Pull request by Anyhowclick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,21 @@ For more details, please visit our [developer portal](https://developer.kyber.ne
...
 2. Run `./tst.sh`
 3. Use `-f` for running a specific test file.
-4. Use `-k` to specify a specific hardfork version. Runs on Istanbul by default.
+4. Use `-k` to specify a specific hardfork version. Default: `Istanbul`
+5. Use `-a` to run both sol4 and sol5 tests. Runs only sol5 tests by default.
 
 ### Example Commands
-`./tst.sh -f ./test/kyberReserve.js`
-`./tst.sh -f ./test/kyberNetworkProxy.js -k petersburg`
+`./tsts.sh` (Run only sol5 tests)
```
Comment (ilanDoron) [inline README.md]: type
tsts

File: test/sol5/kyberMatchingEngine.js [hunk 2]
```diff
@@ -2199,7 +2199,6 @@ contract('KyberMatchingEngine', function(accounts) {
 
     describe("test edge cases for fee. very big / small network fee and very / big small custom fee", async() => {
         //todo: check what happens if fee combination is above 100% or equal 100% or very near 100%
```
Comment (ilanDoron) [inline test/sol5/kyberMatchingEngine.js]: I think can remove this describe. no?
u already tested it?

Comment (Anyhowclick) [reply test/sol5/kyberMatchingEngine.js]: Fee combinations, no. Not yet

Comment (Anyhowclick) [reply test/sol5/kyberMatchingEngine.js]: wait, there was a line change here?

Comment (Anyhowclick) [reply test/sol5/kyberMatchingEngine.js]: Removed
